File: python/word_count.py
def count_word(n:str) -> int:
    if len(n) <= 0:
        raise ValueError("Invalid input string")
    
    word = n.split()
    count = len(word)  
    
    return count    


def main():
    sentence = "My name is Walon-Jalloh"

    count = count_word(sentence)

    print(f'Word count in the sentence is {count}')

main()

File: typescript/src/word_count.ts
function word_count(sentence:string):number {
    let word_list = sentence.split(" ")
    if (word_list.length <= 1){
        return 1
    }

    return word_list.length

}

const sentence = "Walon is crazy and is also a good person";
const count = word_count(sentence)

console.log(`Number of word in the sentence are: ${count}`)